api_app/api/routes/workspaces.py

```diff
@@ -6,18 +6,22 @@
 
 from api.dependencies.database import get_repository
```

Comment:
Can we avoid putting more into this one python file? With airlock it feels like a good candidate to be in it's own file?

Reply:
Moved.